docs/reference.md

```diff
@@ -69,7 +69,7 @@ we encapsulate details about how different element types are handled in
 
 We can use the interpreter mechanism to fold operations with constant operand
 values. The following code snippet demonstrates an idea of the implementation
-for folding `stablehlo::AddOp` with floating-point typed operands:
+for folding `stablehlo::AddOp` with Floating-point typed operands:
```

Comment:
Canonical spelling in the spec doesn't capitalize "Integer", "Floating-point" etc. unless they are at the beginning of a sentence. Let's follow the same convention here and below?